src/lib/proposition.test.ts

```diff
@@ -134,3 +136,47 @@ describe('listPropositions', () => {
     );
   });
 });
+describe('setSupportedBy', () => {
+  it('creates a supportedBy edge', async () => {
+    expect.assertions(1);
+    const prop1 = await createProposition({
+      supportedBy: [],
+      supports: [],
+      text: 'All greeks are mortal',
+    });
+    const prop2 = await createProposition({
+      supportedBy: [],
+      supports: [],
+      text: 'All men are mortal',
+    });
+
+    await setSupportedBy(prop1.id, [prop2.id]);
+    await expect(getProposition(prop1.id)).resolves.toEqual(
+      expect.objectContaining({
+        supportedBy: [prop2.id],
+      })
+    );
+  });
+});
+describe('setSupports', () => {
+  it('creates a supports edge', async () => {
+    expect.assertions(1);
+    const prop1 = await createProposition({
+      supportedBy: [],
+      supports: [],
+      text: 'All men are mortal',
+    });
+    const prop2 = await createProposition({
+      supportedBy: [],
+      supports: [],
+      text: 'All greeks are mortal',
+    });
+
+    await setSupports(prop1.id, [prop2.id]);
+    await expect(getProposition(prop1.id)).resolves.toEqual(
```

Comment:
We should also assert that if we `getProposition(prop2.id)` it has a `supportedBy` containing `prop1.id`.